data/helpers.d/php

```diff
@@ -2,7 +2,7 @@
 
 # Declare the actual php version to use.
 # A packager willing to use another version of php can override the variable into its _common.sh.
-YNH_PHP_VERSION=${YNH_PHP_VERSION:-$YNH_DEFAULT_PHP_VERSION}
+YNH_PHP_VERSION=${YNH_PHP_VERSION:-${YNH_DEFAULT_PHP_VERSION:-7.0}}
```

Comment:
Uh but is this change still needed despite the fix in backup.py ?

Reply:
Yeah --'

https://github.com/YunoHost/yunohost/blob/b0cd37aecad25bacd74c765101473d8ca8150d7d/data/hooks/backup/05-conf_ldap#L7

This line fails the tests, because there is no `YNH_DEFAULT_PHP_VERSION` defined.
We could move `set -eu` right after sourcing helpers, but I think it would be better if no errors are thrown when we sourcing all our helpers.

Reply:
Uh indeed ... Or we could inject the var in hook_exec() directly ... dunno if that's a better design or not ... nvm let's just keep your code :+1: